plugins/techdocs/src/reader/transformers/html/transformer.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import DOMPurify from 'dompurify';
+import { useMemo, useCallback } from 'react';
+
+import { useApi, configApiRef } from '@backstage/core-plugin-api';
+
+import { Transformer } from '..';
```

Comment:
importing from '..' is frowned upon I've learnt 😅 should probably be '../transformer' instead.

Reply:
TIL